src/resources/transactions/transaction.ts

```diff
@@ -0,0 +1,164 @@
+import { NativeCurrencyKey } from '@/entities';
+import {
+  createQueryKey,
+  queryClient,
+  QueryFunctionArgs,
+  QueryFunctionResult,
+} from '@/react-query';
+import { useQuery } from '@tanstack/react-query';
+import {
+  consolidatedTransactionsQueryFunction,
+  consolidatedTransactionsQueryKey,
+} from './consolidatedTransactions';
+import { useAccountSettings } from '@/hooks';
+import { RainbowNetworks, getNetworkObj } from '@/networks';
+import { rainbowFetch } from '@/rainbow-fetch';
+import { ADDYS_API_KEY } from 'react-native-dotenv';
+//import { parseTransaction } from '@/parsers/transactions';
```

Comment:
```suggestion
```

Reply:
should be nit, gonna uncomment that right after, just trying to reduce # of files for review at a time